### DISCUSSION TITLE
Allow us to delete our own issues on OTHER repositories

### DISCUSSION THREAD

**Discussion by ivo95815** [community/community#47088] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It would be grateful if we would have the option to delete our own issues on other repositories that are 90 days old (just think of something). Sometimes I have issues which are no longer relevant or accidentally created and I want to delete them. Hope to see this implemented ASAP, thank you.

**Comment by ghost** [comment]
```
Go
```

**Comment by ghost** [comment]
Alialmrhbi0@gmail.com

**Reply by ghost** [reply]
ا

**Comment by jge162** [comment]
I agree that would be nice.

**Reply by ghost** [reply]
****

**Comment by ghost** [comment]
@ivo95815 @jge162 @Ali-Activity @zackfern @samus-aran @timoyates

**Comment by ivo95815** [comment]


**Reply by ghost** [reply]
@jge162

**Reply by jge162** [reply]
> Who is "Ali-Activity"? Is it a bot or something..?

I believe it’s a bot
